units/commit_queue.go

```diff
@@ -288,22 +292,28 @@ func getModules(ctx context.Context, githubToken string, nextItem *commitqueue.C
 	return modulePRs, modulePatches, false, nil
 }
 
-func getPatchInfo(ctx context.Context, githubToken string, patchDoc *patch.Patch) (string, []patch.Summary, *model.Project, error) {
+func getPatchInfo(ctx context.Context, githubToken string, patchDoc *patch.Patch) (string, []patch.Summary, *model.Project, bool, error) {
```

Comment:
maybe we should return a strut rather than 5 values?

Reply:
returns 4 now.
Could still package in a struct 🤷‍♂️